src/app/@ansyn/ansyn/modules/overlays/overlay-status/effects/overlay-status.effects.ts

```diff
@@ -91,14 +100,20 @@ export class OverlayStatusEffects {
 		filter(Boolean),
 		withLatestFrom(this.store$.select(mapStateSelector), (activeMapId, mapState: IMapState) => MapFacadeService.activeMap(mapState)),
 		filter((activeMap: ICaseMapState) => activeMap && activeMap.data && !activeMap.data.overlay),
-		map((activeMap: ICaseMapState) => new DisableImageProcessing())
+		map(() => new DisableImageProcessing())
 	);
 
 	@Effect()
 	onSelectCase$: Observable<DisableImageProcessing> = this.actions$.pipe(
 		ofType(CasesActionTypes.SELECT_CASE),
 		map(() => new DisableImageProcessing()));
 
+	@Effect({ dispatch: false })
+	onOverlayOutOfBounds: Observable<any> = this.actions$.pipe(
+		ofType(OverlayStatusActionsTypes.ADD_ALERT_MSG),
```

Comment:
Why would we send action ADD_ALERT_MSG is we don't show an alert?
Instead of sending ADD_ALERT_MSG, I suggest to send, in this case, another action like BACK_TO_EXTENT (a new type of action or maybe there is already such an action)